resources/scss/ceres/components/_live-shopping.scss

```diff
@@ -0,0 +1,142 @@
+$live-shopping-padding: 1rem .8rem;
```

Comment:
Bitte nach Möglichkeiten auf bestehende Variablen zurückgreifen oder die Werte in sinnvoller Weise von diesen Variablen ableiten.

https://github.com/plentymarkets/plugin-ceres/blob/stable/resources/scss/_variables.scss

Reply:
done